napari_tiff/napari_tiff_reader.py

```diff
@@ -75,8 +74,9 @@ def tifffile_reader(tif: TiffFile) -> List[LayerData]:
     if nlevels > 1:
         import zarr
         store = tif.aszarr(multiscales=True)
-        group = zarr.hierarchy.group(store=store)
-        data = [arr for _, arr in group.arrays()]  # read-only zarr arrays
+        group = zarr.open_group(store=store, mode='r')
+        # using group.attrs to get multiscales is recommended by cgohlke
```

Comment:
Did we know, in which scenario, the nlevels could be above 1? Is it always zarr?

Reply:
nlevels is above one for pyramidal tiff and tiff variants that contain more than 1 resolution level.
See this explanation from cgolhke: https://forum.image.sc/t/beginners-experiences-on-using-ome-tiff/98077/5?u=psobolewskiphd
It's not related to zarr per se. This plugin just uses the zarr tifffile implementation to read those images.

Reply:
In other words, using zarr isn't strictly required. One can take a level and then use `.asarray()` to get the array. Of course the issue is that most of these WSI have level 0 (full resolution) that are very large, potentially exceeding memory.

Reply:
And of course non-pyramidal images could also be accessed using zarr backend...